gitops/prepare/prepare.go

```diff
@@ -31,12 +31,16 @@ const (
 	BuildBundleConfigMapName = "build-pipelines-defaults"
 	// data key within a configMap that holds the URL to a build bundle
 	BuildBundleConfigMapKey = "default_build_bundle"
+	HACBSBundleConfigMapKey = "hacbs_build_bundle"
+
 	// fallback bundle that will be used in case the bundle resolution fails
 	FallbackBuildBundle = "quay.io/redhat-appstudio/build-templates-bundle:8201a567956ba6d2095d615ea2c0f6ab35f9ba5f"
 	// default secret for app studio registry
 	RegistrySecret = "redhat-appstudio-registry-pull-secret"
 	// Pipelines as Code global configuration secret name
 	PipelinesAsCodeSecretName = "pipelines-as-code-secret"
+	// ConfigMap name for detection hacbs workflow
```

Comment:
I would add your 
```
Note: HACBS detection by configmap is temporary solution, will be
changed to detection based on APIBinding API in KCP environment.
```
note from your PR description in the comment here as well.  In the future folks will not think to look at the PR associated with this change to determine if this is temporary or not.